backend/build.gradle

```
@@ -119,6 +125,25 @@ jacocoTestReport {
     }
 }
 
+task cucumber() {
+    dependsOn assemble, compileTestJava
+    doLast {
+        javaexec {
+            main = "io.cucumber.core.cli.Main"
+            classpath = configurations.cucumberRuntime + sourceSets.main.output + sourceSets.test.output
+            args = [
+                    '--plugin', 'pretty',
+                    '--plugin', 'html:build/reports/cucumber/cucumber-report.html',
+                    '--glue', 'com.woowacourse.gongcheck',
+                    'src/test/resources/features']
+        }
+    }
+}
+
+check {
+    dependsOn(cucumber)
+}
```

```suggestion
check {
    dependsOn cucumber
}
```
통일성있게 문법을 사용하는 것도 좋을 것 같네요
 
build에 의존해도 되는거같은데 왜 check에 의존하고 있는지 설명가능할까요?

---

https://www.baeldung.com/gradle-test-vs-check#check-task

저희 상황에서는 굳이 check가 의존하도록 하지 않아도 될 것 같아요. build로 수정하겠습니다.